src/ResourceManager/Network/Commands.Network/ApplicationGateway/CustomError/SetAzureApplicationGatewayHttpListenerCustomErrorCommand.cs

```diff
@@ -0,0 +1,53 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet("Set", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ApplicationGatewayHttpListenerCustomError"),
+        OutputType(typeof(PSApplicationGatewayCustomError))]
+    public class SetAzureApplicationGatewayHttpListenerCustomErrorCommand : AzureApplicationGatewayHttpListenerCustomErrorBase
+    {
+        [Parameter(
+             Mandatory = true,
+             ValueFromPipeline = true,
+             HelpMessage = "The Application Gateway Http Listener")]
+        public PSApplicationGatewayHttpListener HttpListener { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            base.ExecuteCmdlet();
+
+            var oldCustomError = this.HttpListener.CustomErrorConfigurations.SingleOrDefault(resource => string.Equals(resource.StatusCode, this.StatusCode));
+
+            if (oldCustomError == null)
```

Comment:
why does this matter? If there is no old error the cmdlet should still be able to set the new one.. right?